docker/entrypoint_pg_d/create_extension.sql

```diff
@@ -0,0 +1 @@
+CREATE EXTENSION pg_stat_statements;
```

Comment:
This is missing some volume definitions in `docker-compose.yml`, isn't it? (Was done properly -- I think -- in https://github.com/dalibo/temboard/commit/698269f9ccfa8c969e06bd54b3e54429cbafde4b.)

Reply:
@dlax erf... really not sure of what you mean...